tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -265,3 +275,93 @@ export async function checkJson(vc: any, proofJson: any): Promise<boolean> {
     ).to.be.true;
     return true;
 }
+
+/* 
+    assert linked event
+
+    steps:
+    1. compare event account with signer
+    2. compare event identity with expected identity
+    3. compare event prime identity with expected prime identity
+    4. compare event web3networks with expected web3networks
+    5. check event idGraph LitentryPrimitivesIdentityContext(linkBlock > 0, isActive = true)
+*/
+
+export async function assertLinkedEvent(
+    context: IntegrationTestContext,
+    signer: KeyringPair,
+    events: any[],
+    expectedIdentities: LitentryPrimitivesIdentity[]
+) {
+    assert.isAtLeast(events.length, 1, 'Check assertLinkedEvent error: events length should be greater than 1');
+
+    const eventIdGraph = parseIdGraph(context.sidechainRegistry, events[events.length - 1].data.idGraph, aesKey);
+
+    const keyringType = signer.type;
+    for (let index = 0; index < events.length; index++) {
+        const eventData = events[index].data;
+        // evm address should convert to substrate address before compare
```

Comment:
Would you like to remove this duplicated code ? 🙃

Reply:
thanks.